kubelet/datadog_checks/kubelet/common.py

```diff
@@ -2,19 +2,19 @@
 # All rights reserved
 # Licensed under Simplified BSD License (see LICENSE)
 
+import logging
 import re
 
 from kubeutil import get_connection_info
 
 from datadog_checks.base.utils.tagging import tagger
 
+log = logging.getLogger(__name__)
+
 try:
     from containers import is_excluded as c_is_excluded
 except ImportError:
     # Don't fail on < 6.2
-    import logging
-
-    log = logging.getLogger(__name__)
```

Comment:
LEt's revert that and provide the check's logger in the `get_prometheus_url` parameters

Reply:
well, no then 
![image](https://github.com/DataDog/integrations-core/assets/1266346/8aea5d97-3c52-4acb-996e-34b1a62e0262)